src/EFCore.PG/Query/Expressions/Internal/PgBinaryExpression.cs

```diff
@@ -135,6 +135,7 @@ protected override void Print(ExpressionPrinter expressionPrinter)
                     PgExpressionType.JsonExists => "?",
                     PgExpressionType.JsonExistsAny => "?|",
                     PgExpressionType.JsonExistsAll => "?&",
+                    PgExpressionType.JsonValueForKeyAsText => "->>",
```

Comment:
I realize this isn't necessary now and I'll make sure it's removed before the PR merges.